go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/ElrondNetwork/elrond-go-core v1.1.19
 	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
 	github.com/ElrondNetwork/elrond-go-logger v1.0.7
-	github.com/ElrondNetwork/elrond-vm-common v1.3.15-rc2
+	github.com/ElrondNetwork/elrond-vm-common v1.3.15-rosetta2
```

Comment:
Perhaps we should create a tag `v1.3.15-rc3` that is an alias to `v1.3.15-rosetta2`?

Reply:
Fixed to reference `v1.3.20`.